src/modules/core/components/PermissionRequiredInfo/PermissionRequiredInfo.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useIntl, defineMessages, FormattedMessage } from 'react-intl';
+import { ColonyRole } from '@colony/colony-js';
+
+import Heading from '~core/Heading';
+import PermissionsLabel from '~core/PermissionsLabel';
+import Icon from '~core/Icon';
+import styles from './PermissionRequiredInfo.css';
+import { Tooltip } from '~core/Popover';
+
+const MSG = defineMessages({
+  title: {
+    id: 'PermissionRequiredInfo.title',
+    defaultMessage: 'You need the following permission(s)',
+  },
+  sectionLabel: {
+    id: 'PermissionRequiredInfo.sectionLabel',
+    defaultMessage: 'Permission',
+  },
+  tooltipTextFirstParagraph: {
+    id: 'PermissionRequiredInfo.tooltipTextFirstParagraph',
+    defaultMessage: `Colony has six permission classes which can be assigned 
+      to any Ethereum address: Funding, Administration, Arbitration, 
+      Architecture, Root, and Recovery.`,
+  },
+  tooltipTextSecondParagraph: {
+    id: 'PermissionRequiredInfo.tooltipTextSecondParagraph',
+    defaultMessage: `Permissions are assigned at the domain level. 
+      If an account is granted a permission in a domain, 
+      it also has that same permission in all of that 
+      domain’s subdomains.`,
+  },
+});
+
+interface Props {
+  /** Array with roles required */
+  requiredRoles: ColonyRole[];
+}
+
+const displayName = 'PermissionRequiredInfo';
+
+const PermissionRequiredInfo = ({ requiredRoles }: Props) => {
+  const { formatMessage } = useIntl();
+  const tooltipText = (
+    <>
+      <p className={styles.tooltipText}>
+        <FormattedMessage {...MSG.tooltipTextFirstParagraph} />
+      </p>
+      <p className={styles.tooltipText}>
+        <FormattedMessage {...MSG.tooltipTextSecondParagraph} />
+      </p>
+    </>
+  );
+
+  return (
+    <div>
+      <Heading
+        appearance={{ size: 'normal', weight: 'bold' }}
+        text={MSG.title}
+        className={styles.title}
+      />
+      <div className={styles.requiredPermissionSection}>
+        <div className={styles.section}>
+          <p className={styles.sectionLabel}>
+            <FormattedMessage {...MSG.sectionLabel} />
+          </p>
+          <Tooltip
+            darkTheme
+            content={tooltipText}
+            placement="right"
+            popperProps={{
+              modifiers: [
+                {
+                  name: 'offset',
+                  options: {
+                    offset: [120, 12],
+                  },
+                },
+              ],
+            }}
+          >
+            {({ close, open, ref }) => (
+              <div
+                ref={ref}
+                onMouseEnter={open}
+                onMouseLeave={close}
+                className={styles.labelIcon}
+              >
+                <Icon name="question-mark" title="" />
```

Comment:
Always set a icon's title.

In this case you can just pass in the `MSG.title` message descriptor